python/server.py

```diff
@@ -30,21 +30,14 @@
 PLAID_COUNTRY_CODES = os.getenv('PLAID_COUNTRY_CODES', 'US').split(',')
 
 
-def empty_to_none(field):
-  value = os.getenv(field)
-  if value is None or len(value) == 0:
```

Comment:
if len(value) is zero, then developer shouldn't have specified an empty redirect uri in the first place

Reply:
do you think that if i run a command with PLAID_REDIRECT_URI= <command> that it should be invalid?

Reply:
yep, specifying an empty redirect uri for oauth is not valid